plugins/ssr/ssrparser.mlg

```diff
@@ -802,6 +797,20 @@ let reject_ssrhid strm =
 
 let test_nohidden = Pcoq.Gram.Entry.of_parser "test_ssrhid" reject_ssrhid
 
+let rec reject_binder crossed_paren k s =
```

Comment:
Can you write a comment saying what this does?

Reply:
Should detect (id: or (Id:=

Reply:
Again, you don't need to put the comments here, they would be more useful in the code IMHO.